main.go

```diff
@@ -42,15 +43,18 @@ import (
 func main() {
 	LogLady.Info("Starting Nancy")
 
+	config := configuration.Config{}
+
 	if len(os.Args) > 1 && os.Args[1] == "iq" {
 		LogLady.Info("Nancy parsing config for IQ")
-		config, err := configuration.ParseIQ(os.Args[2:])
+		config.Type = "iq"
```

Comment:
This `setting of type` on the shared config object feels weird to me. Maybe make a "constructor" (yes i know golang doesn't have those) but basically to create a config you must pass a type of config. 

Something like 
```
func NewConfig(string typeOfConfig) *Config{
    config := configuration.Config{}
    //maybe some validation that "iq" or "ossindex" was passed??
    config.Type = typeOfConfig
    return config
}
```

Reply:
That's a great suggestion. I sorta just jammed this in here to get it working. I'll noodle on that!

Reply:
I think I got this in: 37493b2

Went a lil further, it only accepts an "enum" of sorts, in my next pass I can add some logic that returns an error if you send in the wrong one.